### PR TITLE
numa:increase initial memory size since the old is boundary value on power

### PR DESCRIPTION
**Pull request by MiriamDeng**
ID:1953871 
Signed-off-by: MiriamDeng <mdeng@redhat.com>

### CODE REVIEW

**Comment by MiriamDeng** [comment]
(1/3) repeat1.Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.ppc64le.io-github-autotest-qemu.numa.numa_consistency: PASS (84.87 s)
 (2/3) repeat2.Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.ppc64le.io-github-autotest-qemu.numa.numa_consistency: PASS (103.22 s)
 (3/3) repeat3.Host_RHEL.m8.u5.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.ppc64le.io-github-autotest-qemu.numa.numa_consistency: PASS (78.01 s)

**Comment by MiriamDeng** [comment]
@PaulYuuu can you help to review it ? Thanks.

File: qemu/tests/numa_consistency.py [hunk 1]
```diff
@@ -51,6 +52,8 @@ def get_vcpu_used_node(numa_node_info, vcpu_thread):
     params['backend_mem'] = "memory-backend-ram"
     params['use_mem'] = "no"
     params['size_mem'] = "1024M"
+    if arch.ARCH in ('ppc64', 'ppc64le'):
+        params['size_mem'] = "4096M"
```
Comment (PaulYuuu) [inline qemu/tests/numa_consistency.py:56]: Could you explain the reason in the commit message?

Comment (MiriamDeng) [reply qemu/tests/numa_consistency.py:56]: The original one is too small for Power, so preferring to have a practical and reasonable size, thanks.

**Comment by MiriamDeng** [comment]
@maxujun can you help to review it ? Thanks

**Comment by maxujun** [comment]
commit fe135682b6b946a6ff7ff54344945d6cda282a2a (HEAD -> master)
Author: MiriamDeng <mdeng@redhat.com>
Date:   Fri Apr 30 18:25:48 2021 +0800

    numa:adjust initial memory size while doing numa consistent test on ppc need to increase memory since the old is criticality value on power Signed-off-by: MiriamDeng <mdeng@redhat.com>

Commit header is too long,I think. Others is ok.

**Comment by MiriamDeng** [comment]
Updated it again, could you help to review again ? @maxujun @PaulYuuu  Thanks.

**Comment by maxujun** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.numa.numa_consistency: PASS (141.18 s)
LGTM, Ack

**Pull request merged**